backend/helpers/emailTemplates.js

```diff
@@ -0,0 +1,13 @@
+function sendTemplates(name, link){
```

Comment:
This is not what we are looking for. For this we require a function for every template.